scripts/build-release-images.sh

```diff
@@ -12,11 +12,19 @@ SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
 source "${SCRIPT_DIR}/release-var.sh"
 source "${SCRIPT_DIR}/common.sh"
 
+# add qemu-user-static
+sudo podman run --rm --privileged docker.io/multiarch/qemu-user-static --reset -p yes
```

Comment:
What does that do? Also, we should try to not use images on Docker hub.

Reply:
Hi @0xFelix Here we are trying to use qemu-user-emulation to build multiarch images on x86 native platform. i was taken this from here https://github.com/kubevirt/kubevirt/blob/add278ad94ca13c112b8da83f41160f7371501b5/hack/builder/build.sh#L19 to cross building on x86 platform.

Reply:
Is the qemu-user-static image available on quay?

Reply:
I don't see where this is registered as a runner for foreign binaries though? Does it happen in the container? Also, why not make it conditional like in the file you've linked?

Reply:
> Is the qemu-user-static image available on quay?

No its not there in quay.

Reply:
> I don't see where this is registered as a runner for foreign binaries though? Does it happen in the container? Also, why not make it conditional like in the file you've linked?
No, it wont happen in container. sure i will make a check before running it.

Reply:
> I don't see where this is registered as a runner for foreign binaries though? Does it happen in the container? Also, why not make it conditional like in the file you've linked?

No, it wont happen in container. sure i will make a check before running it.

Reply:
> I don't see where this is registered as a runner for foreign binaries though? Does it happen in the container? Also, why not make it conditional like in the file you've linked?

Hi @0xFelix added condition to install if not enabled.